hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/EventResponseTemplate.java

```diff
@@ -202,4 +203,10 @@ void IMapInvalidation(@Nullable Data key, @Since("1.4") String sourceUuid,
     @EventResponse(EventMessageConst.EVENT_IMAPBATCHINVALIDATION)
     void IMapBatchInvalidation(List<Data> keys, @Since("1.4") List<String> sourceUuids,
                                @Since("1.4") List<UUID> partitionUuids, @Since("1.4") List<Long> sequences);
+
+    /**
```

Comment:
you may indicate in the javadoc description that the all partition mappings will be covered (it is a complete list matching the configured partition count) in the returned event.